lib/active_record/immutable.rb

```diff
@@ -0,0 +1,49 @@
+module ActiveRecord
+  module Immutable
+    extend ActiveSupport::Concern
+
+    FIELD_NAME_IDENTIFIER_IMMUTABLE = 'previous_version_id'.freeze
```

Comment:
What do you think about renaming this to `PREVIOUS_VERSION_ID` or `PREVIOUS_VERSION_ID_COLUMN_NAME`?

Reply:
About the `.freeze`, I think we should remove it and add the `# frozen_string_literal: true` on the top of the file.

Reply:
In private conversation, the name was set to `DEFAULT_PREVIOUS_VERSION_ID_COLUMN_NAME`